### PR TITLE
Issue #14

### PR DESCRIPTION
**Pull request by judytafilipkowska**
Added dark theme toggle

### CODE REVIEW

**Review by marcnjaramillo — COMMENTED** [review]
This looks good. Thanks for working on this feature!

File: index.html [hunk 1]
```diff
@@ -12,6 +12,9 @@
     {% include description.html %}
     
     {% include projects.html %}
+
+    {% include education.html %}
```
Comment (prakharrathi25) [inline index.html:16]: Do we need the education section? This wasn't discussed in the issue. It seems like a feature from an old MLH portfolio website which is redundant now because we are not making an individual portfolio website but instead we are making a portfolio for the whole pod. I think it might be good to remove it.

File: projects/jobhunter.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (prakharrathi25) [inline projects/jobhunter.md:1]: This file also seems extra for the dark mode PR because we already have three projects to test on. This project  might just lead to a merge conflict later.

File: _sass/navbar.scss [hunk 3]
```diff
@@ -1,36 +1,40 @@
...
-    padding-top: 10px;
+  width: 100%;
+  background-color: #133667;
+  font-family: "Poppins", serif;
+  font-weight: 700 !important;
+  padding-top: 10px;
 }
 
 .nav-content .logo-touch-target {
-    display: inline-flex;
```
Comment (prakharrathi25) [inline _sass/navbar.scss:10]: This might conflict with @RishiDesai17 and @yahya-cloud navbar PR. Maybe check in with them once?

File: _includes/education.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<h2>Education</h2>
```
Comment (prakharrathi25) [inline _includes/education.html:1]: Same comment as before. Education section is extra and not needed for a pod portfolio. This was not mentioned in the issue either.

File: _data/projects.yml [hunk 5]
```diff
@@ -1,14 +1,9 @@
-- title: Online Shopping App
```
Comment (prakharrathi25) [inline _data/projects.yml:1]: Is there a reason you changed the list of projects? This is problematic because this will conflict with other changes and it seems like an unnecessary change.

File: _data/fellows.yml [hunk 6]
```diff
@@ -1,15 +1,7 @@
-- name: Will
```
Comment (prakharrathi25) [inline _data/fellows.yml:1]: This is also not required to be changed. It will also lead to a bunch of conflicts. Please fetch upstream before pushing to your fork.

File: _data/education.yml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+- course: Prep Fellow
```
Comment (prakharrathi25) [inline _data/education.yml:1]: This file is also not required. reasons stated already.

**Review by prakharrathi25 — CHANGES_REQUESTED** [review]
The dark mode seems great but I have added a bunch of comments which would help you fix your merge conflicts. Most of it is because of redundant files and changes which are not required for your feature. Happy to discuss the comments and the PR with you further.

**Comment by marcnjaramillo** [comment]
This pull request is outdated. Closing.
